lib/Sema/CSSimplify.cpp

```diff
@@ -3327,6 +3327,10 @@ ConstraintSystem::simplifyBridgingConstraint(Type type1,
                                              Type type2,
                                              TypeMatchOptions flags,
                                              ConstraintLocatorBuilder locator) {
+  // There's no bridging without ObjC interop.
+  if (!TC.Context.LangOpts.EnableObjCInterop)
```

Comment:
In practice, we do. Should this check be moved up to the places that introduce bridging constraints?